hbase-protocol-shaded/src/main/protobuf/client/Client.proto

```diff
@@ -274,6 +274,7 @@ message Scan {
   }
   optional ReadType readType = 23 [default = DEFAULT];
   optional bool need_cursor_result = 24 [default = false];
+  optional uint32 limit = 25;
```

Comment:
I thought this is duplicate with ScanRequest protobuf? We already have one field to means "limit".

Reply:
Do we need to add a new filed to means "scan limit"?

Reply:
yes, duplicate field may confuse user